.github/workflows/release.yaml

```diff
@@ -181,6 +181,26 @@ jobs:
         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
         restore-keys: |
           ${{ runner.os }}-go-
+    - name: Install pnpm
+      uses: pnpm/action-setup@v3
+      with:
+        version: 8
+        run_install: false
+    - name: Install nodejs
+      uses: actions/setup-node@v4
+      with:
+        node-version: "20.12.1"
+        cache: "pnpm"
+        cache-dependency-path: "**/pnpm-lock.yaml"
+    - name: Build UI
+      env:
+        VERSION: ${{ github.ref_name }}
+      working-directory: ./ui
+      run: |
+        pnpm install
+        NODE_ENV=production pnpm run build
+        rm -rf ../internal/api/ui
+        mv build ../internal/api/ui
```

Comment:
This builds the UI and copies it into a location where it will be included in the CLI build.

This grows the size of the CLI by ~25M, but that is the price for serving the UI from a local server built into the CLI.

Reply:
1. Shouldn't "unstable" include this as well?
2. I think we now build the UI twice, once in the image and another time for the CLI. It may be better to instead extract this data from the image?

Reply:
Don't t recall off the top of my head, but thought we were not publishing unstable CLI nightly. If I'm wrong (quite possible) then I agree. I will double check this when able.

Reply:
Ok. Glad you pointed this out. We _were_ publishing the nightly CLI builds. Should be fixed now. I also slightly cleaned up _how_ we were doing those, because we weren't using a matrix-based strategy like we were for the official one.